dvc/repo/remove.py

```diff
@@ -11,7 +18,25 @@
 
 @locked
 def remove(self: "Repo", target: str, outs: bool = False):
-    stages = self.stage.from_target(target)
+    try:
+        stages = self.stage.from_target(target)
+    except (StageNotFound, StageFileDoesNotExistError) as e:
+        # If the user specified a tracked file as a target instead of a stage,
+        # e.g. `data.csv` instead of `data.csv.dvc`,
+        # give a more helpful error message.
+        try:
+            self.stage.from_target(target + DVC_FILE_SUFFIX)
```

Comment:
Since this only shows a hint, could we check if `target + DVC_FILE_SUFFIX` file exists?

Reply:
I think I have to leave this question to your team, I don't know dvc well enough. Maybe ping someone else to chime in. Can you find a concrete, plausible scenario where it makes a difference, and explain why the result of either approach is better?

Reply:
I agree with @daavoo , we should just check if it exists here. Or not check at all even and just show the hint (we do the same in checkout).